cpp/test/sparse/neighbors/cross_component_nn.cu

```diff
@@ -14,13 +14,15 @@
  * limitations under the License.
  */
 
-// XXX: We allow the instantiation of fused_l2_nn here:
-// raft::linkage::FixConnectivitiesRedOp<value_idx, value_t> red_op(colors.data(), params.n_row);
-// raft::linkage::connect_components<value_idx, value_t>(
+// XXX: We allow the instantiation of masked_l2_nn here:
+// raft::linkage::FixConnectivitiesRedOp<value_idx, value_t> red_op(params.n_row);
+// raft::linkage::cross_component_nn<value_idx, value_t>(
 //   handle, out_edges, data.data(), colors.data(), params.n_row, params.n_col, red_op);
 //
 // TODO: consider adding this to libraft.so or creating an instance in a
 // separate translation unit for this test.
+//
+// TODO: edge case testing
 #undef RAFT_EXPLICIT_INSTANTIATE_ONLY
```

Comment:
I don't think this is needed. masked_l2_nn is not pre-compiled.

Reply:
Done